test/support/factory.ex

```diff
@@ -17,7 +17,7 @@ defmodule Livebook.Factory do
     %Livebook.Hubs.Fly{
       id: "fly-foo-bar-baz",
       hub_name: "My Personal Hub",
-      hub_color: "#FF00FF",
+      hub_emoji: "🪽",
```

Comment:
```suggestion
      hub_emoji: "💸",
```

the current one doesn't render here at least on my machine :)

Reply:
Renders in gmail though 😄